dotcom-rendering/src/web/components/Snap.tsx

```diff
@@ -5,6 +5,7 @@ const snapStyles = css`
 	overflow-y: hidden;
 	position: relative;
 	display: flex;
+	flex-direction: column;
```

Comment:
| Before | After |
|--------|-------|
| ![image](https://github.com/guardian/dotcom-rendering/assets/19683595/e8f244dd-cf69-4f78-854b-f405165cbae3) | ![image](https://github.com/guardian/dotcom-rendering/assets/19683595/0f18dde8-ade4-4dfb-96c8-170f602c424e) |

Reply:
Reverted this change. This was fixed by @HarryFischer in the thrasher